deploy-release.sh

```diff
@@ -13,13 +13,19 @@ fi
 gcloud config set project $PROJECT_ID
 
 # Submit a new build to Cloud Build
-BUILD_ID=$(gcloud builds submit --config cloudbuild.yaml . --async --format="value(id)" --region $REGION)
```

Comment:
removing the async here means that the script stopping will stop the build. Please undo this change. The rest looks good.